numpy/core/fromnumeric.py

```diff
@@ -63,10 +82,46 @@ def _wrapfunc(obj, method, *args, **kwds):
     # in its class, but its signature is not identical to that
     # of NumPy's. This situation has occurred in the case of
     # a downstream library like 'pandas'.
-    except (AttributeError, TypeError):
+    except (AttributeError, TypeError) as e:
```

Comment:
Actually, would it be alright to keep this function?  I just realized that I used it in both `wrapfunc` and `wrapfunc2`, so removing it would mean that I would have to duplicate the code in four different places (two methods, both for `AttributeError` and `TypeError`)